test/SILOptimizer/remove_unused_global_vars_wmo.swift

```diff
@@ -0,0 +1,27 @@
+// RUN: %target-swift-frontend -O -wmo -emit-sil %s | %FileCheck %s
```

Comment:
You could probably combine these into one test with two RUN: lines. You can use the --check-prefix flag to FileCheck to change the prefix from CHECK to something else, like CHECK-WMO.